platform/features/layout/src/LayoutController.js

```diff
@@ -58,14 +58,14 @@ define(
                 return copy;
             }
 
-            // Compute panel positions based on the layout's object model
-            function lookupPanels(ids) {
+            /**
+             * Compute panel positions based on the layout's object model.
+             * Defined as member function to facilitate testing.
+             * @private
+             */
+            LayoutController.prototype.layoutPanels = function layoutPanels (ids) {
```

Comment:
Ah, whoops, this was by accident. Thanks for picking it up. I promoted an inner function to a member function so that I could use it in testing, but modified it in place by mistake. With long constructors it can be difficult to know whether you're currently inside or outside of the constructor.